README.md

```diff
@@ -79,7 +79,7 @@
   This section will guide you through using the default setup, which does sufficiently implement the Double Submit Cookie Pattern. If you'd like to customise the configuration, see the <a href="#configuration">configuration</a> section.
 </p>
 <p>
-  You will need to be using <a href="https://github.com/expressjs/cookie-parser">cookie-parser</a> and the middleware should be registered before Double CSRF. This utility will set a cookie containing a hash of the csrf token and provide the non-hashed csrf token so you can include it within your response.
+  You will need to be using <a href="https://github.com/expressjs/cookie-parser">cookie-parser</a> and the middleware should be registered before Double CSRF. This utility will set a cookie containing both the csrf token and a hash of the csrf token and provide the non-hashed csrf token so you can include it within your response.
```

Comment:
I think this section should just specify what the cookie contains, i.e.:

    This utility will set a cookie containing a concatenation of the raw csrf token and the hashed csrf token separated by a `|`

And we should update the documentation for `generateToken`/`req.csrfToken` to show the different cases of overwrite.

Reply:
Pushed a commit with something similar to the suggestions, It also has a new generateToken section that explains how it works and what overwrite does